qpc/utils.py

```diff
@@ -382,3 +382,29 @@ def extract_json_from_tar(fileobj_content, print_pretty=True):
     if print_pretty:
         return pretty_print(json_data)
     return json_data
+
+
+def create_tar_buffer(files_data):
+    """Gernerate a file buffer based off a dictionary."""
```

Comment:
Change `gernerate` to `generate` and also maybe say what the param files_data is and what is returned?

Reply:
😬

Reply:
Thanks Ashley.  That 2nd part was really touching